### PR TITLE
update kfserving website for 0.4.1 release

### PR DESCRIPTION
**Pull request by animeshsingh**
for kf1.2

### CODE REVIEW

**Comment by animeshsingh** [comment]
cc @yuzisun

File: content/en/docs/components/serving/kfserving.md [hunk 1]
```diff
@@ -18,14 +18,14 @@ You can use KFServing to do the following:
 
 * Enable a simple, pluggable, and complete story for your production ML inference server by providing prediction, pre-processing, post-processing and explainability out of the box.
 
-Our strong community contributions help KFServing to grow. We have a Technical Steering Committee driven by Google, IBM, Microsoft, Seldon, and Bloomberg. [Browse the KFServing GitHub repo](https://github.com/kubeflow/kfserving) to give us feedback!
+Our strong community contributions help KFServing to grow. We have a Technical Steering Committee driven by AWS, IBM, Nvidia, Seldon, and Bloomberg. [Browse the KFServing GitHub repo](https://github.com/kubeflow/kfserving) to give us feedback!
```
Comment (8bitmp3) [inline content/en/docs/components/serving/kfserving.md:21]: Just a suggestion - official unofficial names maybe:

```suggestion
Our strong community contributions help KFServing to grow. We have a Technical Steering Committee driven by Amazon Web Services (AWS), IBM Cloud, NVIDIA, Seldon, and Bloomberg. [Browse the KFServing GitHub repo](https://github.com/kubeflow/kfserving) to give us feedback!
```

(Had to go [here](https://s22.q4cdn.com/364334381/files/doc_financials/2020/ar/2020-nvidia-annualreport-content-r25-web-144dpi-combined.pdf) to check "NVIDIA" 🤣 Their site uses all caps everywhere, I think)

File: content/en/docs/components/serving/kfserving.md [hunk 2]
```diff
@@ -18,14 +18,14 @@ You can use KFServing to do the following:
...
 
-Kubeflow 1.1 includes KFServing v0.3, where the focus has been on providing more stability by doing a major move to KNative v1 APIs. Additionally, we added GPU support for PyTorch model servers, and pickled model format support for SKLearn. There were other enhancements to routing, payload logging, including bug fixes etc., details of which can be found [here](https://github.com/kubeflow/kfserving/releases/tag/v0.3.0).
+Kubeflow 1.2 includes KFServing v0.4.1, where the focus has been on enabling KFServing on OpenShift, and additionally providing more features like adding batcher module as sidecar, Triton inference server renaming and integrations, upgrading Alibi explainer to 0.4.0, allowing customized url paths on data plane, updating logger to CloudEvents V1 protocol etc. Additionally minimum Istio version has been moved to Istio v1.3.1. More details can be found [here](https://github.com/kubeflow/kfserving/releases/tag/v0.4.0) and [here](https://github.com/kubeflow/kfserving/releases/tag/v0.4.1)
```
Comment (8bitmp3) [inline content/en/docs/components/serving/kfserving.md:28]:
```suggestion
Kubeflow 1.2 includes KFServing v0.4.1, where the focus has been on enabling KFServing on OpenShift and additionally providing more features, such as adding batcher module as sidecar, Triton inference server renaming and integrations, upgrading Alibi explainer to 0.4.0, and allowing customized URL paths on data plane, updating logger to CloudEvents V1 protocol. Additionally, the minimum Istio version is now v1.3.1. More details can be found [here](https://github.com/kubeflow/kfserving/releases/tag/v0.4.0) and [here](https://github.com/kubeflow/kfserving/releases/tag/v0.4.1)
```

WDYT

**Comment by animeshsingh** [comment]
thanks. addressed some comments

**Review by yuzisun — COMMENTED** [review]
/lgtm

**Pull request merged**